.gitignore

```diff
@@ -1,5 +1,6 @@
 # Go vendor
 /vendor
+/github.com
```

Comment:
Any reason why to add this ignore?

Reply:
`go run github.com/djencks/gen-crd-api-reference-docs@5319c98b89a5438b096aeccb0049a5f6c75543a6` appears to download to /github.com, so I wanted to avoid committing it. I'm surprised at this behavior of `go run`, but don't know any better.